gpt_index/indices/base.py

```diff
@@ -601,7 +600,9 @@ def save_to_string(self, **save_kwargs: Any) -> str:
         out_dict = self.save_to_dict(**save_kwargs)
         return json.dumps(out_dict, **save_kwargs)
 
-    def save_to_disk(self, save_path: str, **save_kwargs: Any) -> None:
+    def save_to_disk(
+        self, save_path: str, encoding: str = "ascii", **save_kwargs: Any
```

Comment:
should encoding be utf-8 by default?

Reply:
We still save to disk using ASCII by default, unless people say ensure_ascii=False 

Really up to you